src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Hmm seems like the first action item for that is to document all the cases where we use `chrome.*` APIs and see if those have a polyfill. With sweeping cross-platform plays like that, we lock ourselves in to only use the APIs which exist on all platforms, which I'm cool with if @tshaddix is, but I would be more comfortable with that being handled in a larger PR that gets tested with our own Chrome extension and others just to make sure everything is peachy.